examples/css/index.html

```diff
@@ -9,7 +9,7 @@
     </head>
     <body data-mistica-skin="movistar" data-mistica-color-scheme="light">
         <div id="app" class="mistica-responsive-layout">
-            <h1 class="mistica-text-title4">Hello mistica css</h1>
+            <h1 class="mistica-text-title4">Mistica css</h1>
```

Comment:
```suggestion
            <h1 class="mistica-text-title4">Mística CSS</h1>
```

Just a stupid thing xD but always use "tick" to "Mística" name

and CSS i think should be in uppercase because is an acronym